godtools/App/Flows/Flow/FlowStep.swift

```diff
@@ -212,7 +212,7 @@ enum FlowStep {
     case didCreateSessionFromCreatingToolScreenShareSession(result: Result<WebSocketChannel, TractRemoteSharePublisherError>)
     case cancelTappedFromCreateToolScreenShareSessionTimeout
     case acceptTappedFromCreateToolScreenShareSessionTimeout
-    case shareQRCodeTappedFromToolScreenShareSession
+    case shareQRCodeTappedFromToolScreenShareSession(shareUrl: String)
```

Comment:
Hey @levieggertcru, would you prefer to regenerate the shareUrl for the QR code view instead of passing it?